packages/embeds/embed-core/src/embed-iframe.ts

```diff
@@ -341,14 +341,8 @@ function keepParentInformedAboutDimensionChanges() {
     }
 
     const mainElementStyles = getComputedStyle(mainElement);
-    const contentHeight =
-      mainElement.offsetHeight +
-      parseInt(mainElementStyles.marginTop) +
-      parseInt(mainElementStyles.marginBottom);
```

Comment:
Oh, I should have used parseFloat.
@LartyHD Thanks. But it I don't think .height and .width consider margin.

Reply:
@LartyHD Thanks. Also, I couldn't replicate the scroll scenario you screenshotted. Is there a page where I can see it in action?

Reply:
What is the motivation behind using .height instead of .offsetHeight. I think offsetHeight includes padding and border as well which we need.

Reply:
You can check it out here: [https://nobletary-dqm7oxxac-larsartmann.vercel.app/en/appointment](https://nobletary-dqm7oxxac-larsartmann.vercel.app/en/appointment)
(The current version overflows to the footer, but that can be ignored)

Reply:
I hope that clarifies things

Reply:
Adding an `overflow-y: hidden` could possibly also be an alternative solution.

Reply:
The other problem is that it only happens sometimes.
I recorded a video, at second 14 I manage to make the sidebar appear.
https://user-images.githubusercontent.com/23587853/224332473-f7b33baa-5149-4393-9cfb-cb89a86825c0.mov

Reply:
@LartyHD Thanks. I could replicate the issue now 🙏

My concern was that .height won't include padding and border. But that happens in the default behaviour of box-sizing(which is content-box). Tailwind explicitly set `box-sizing` to `border-box` in which case it includes padding and border.

So, in this case .height is better than offsetHeight in giving the correct value of height.